oozie/src/main/java/org/apache/falcon/service/LogMoverService.java

```diff
@@ -44,7 +44,8 @@
     public static final String ENABLE_POSTPROCESSING = StartupProperties.get().
             getProperty("falcon.postprocessing.enable");
 
-    private BlockingQueue<Runnable> blockingQueue = new ArrayBlockingQueue<>(50);
+    private BlockingQueue<Runnable> blockingQueue = new ArrayBlockingQueue<>(Integer.parseInt(
+            StartupProperties.get().getProperty("falcon.logMoveService.blockingQueue")));
```

Comment:
Please provide a default value, in case of absence of the property in the startup.properties.